docs/fetchingandsubscribingdata.md

```diff
@@ -0,0 +1,21 @@
+# Fetching data with apollo and subscribing to updates
```

Comment:
I think much of this can (soon) go into the client documentation, that it will include code snippets, and it will be sibling to docs about the general use of the API to fetch data/entities.  Alchemy-specific stuff would of course remain in the Alchemy docs.